swarm/network/kademlia_test.go

```diff
@@ -864,34 +863,16 @@ func TestEachBin(t *testing.T) {
 		"81968a2d8fb39114342ee1da85254ec51e0608d7f0f6997c2a8354c260a71009",
 	}
 
-	//construct the peers and the kademlia
+	//create the pivot's kademlia
 	addr := common.FromHex(pivotAddr)
-	addrs = append(addrs, pivotAddr)
-
 	k := NewKademlia(addr, NewKadParams())
 
-	as := make([][]byte, len(addrs))
-	for i, a := range addrs {
-		as[i] = common.FromHex(a)
-	}
-
-	for _, a := range as {
-		if bytes.Equal(a, addr) {
-			continue
-		}
-		p := &BzzAddr{OAddr: a, UAddr: a}
-		if err := k.Register(p); err != nil {
-			t.Fatal(err)
-		}
+	//construct the peers and the kademlia
+	for _, a := range addrs {
```

Comment:
only `On` the ones that are in the left (connected) part of the kademlia not the whole addressbook (right side)